Brightspace/insights-adoption-dashboard
# Remove Legend Interactions

Functional
 - Manual testing - Demo Env
   - Cursor is no longer a hand when hovering
   - Can't tab into legend items
   - Can't use H to header jump to legend (screen reader)
   - SVG color icons appear in print 
   
Demo testing is sufficient IMO since this is strictly a browser platform change

UX
 - Legend bubbles are slightly smaller to match font height

Docs N/A
Security N/A
